www/src/components/layout/layout-default.js

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+import { SkipNavLink } from "@reach/skip-nav"
+import styled, { css } from "react-emotion"
+
+import TopBanner from "./top-banner"
+import PageHeading from "./page-heading"
+import Navigation from "../navigation"
+import MobileNavigation from "../navigation-mobile"
+
+import presets, { colors } from "../../utils/presets"
+
+// Import Futura PT typeface
+import "../../fonts/Webfonts/futurapt_book_macroman/stylesheet.css"
+import "../../fonts/Webfonts/futurapt_bookitalic_macroman/stylesheet.css"
+import "../../fonts/Webfonts/futurapt_demi_macroman/stylesheet.css"
+import "../../fonts/Webfonts/futurapt_demiitalic_macroman/stylesheet.css"
+// Other fonts
+import "typeface-spectral"
+
+const Content = styled("div")`
+  padding-top: ${presets.bannerHeight};
+  height: 200vh;
+`
+
+const StyledSkipNavLink = styled(SkipNavLink)`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  position: absolute;
+  z-index: 100;
+  font-size: 0.85rem;
+
+  &:focus {
```

Comment:
Not a problem, but `emotion` doesn't require the `&` unless you're trying to override specificity.

Reply:
OK